diff --git a/simms/casasm.py b/simms/casasm.py
index 9bb6e96..5216785 100644
--- a/simms/casasm.py
+++ b/simms/casasm.py
@@ -32,7 +32,7 @@ def wgs84_2xyz(pos_wgs84):
 def enu2xyz (refpos_wgs84,enu):
     """ converts xyz0 + ENU (Nx3 array) into xyz """
     refpos = me.measure(refpos_wgs84,'itrf')
-    lon,lat,rad = [ refpos[x]['value'] for x in 'm0','m1','m2' ]
+    lon,lat,rad = [ refpos[x]['value'] for x in ('m0','m1','m2') ]
     xyz0 = rad*np.array([math.cos(lat)*math.cos(lon),math.cos(lat)*math.sin(lon),math.sin(lat)])
     # 3x3 transform matrix. Each row is a normal vector, i.e. the rows are (dE,dN,dU)
     xform = np.array([
